comictaggerlib/taggerwindow.py

```diff
@@ -1620,7 +1634,8 @@ def copy_tags(self) -> None:
         ca_list = self.fileSelectionList.get_selected_archive_list()
         has_src_count = 0
 
-        src_style = self.load_data_style
+        # TODO: Take first read style for now (make a better system later)
+        src_style = self.load_data_styles[0]
```

Comment:
This could get quite complex I think. An initial idea was to have a window for copying tags?

Reply:
I was thinking of a window so you can choose more independently. I've done the overlayed result so an independent copy tag PR could be a backburner.